src/stores/prompts.js

```diff
@@ -71,28 +71,13 @@ export const usePromptStore = defineStore('prompts', {
       this._isLoading = false
     },
 
-    async fetchPromptsByYear(year) {
```

Comment:
Yup. Fair. Unlikely to be necessary. 

Found it. 
  ```
if (router.currentRoute.value.params.year) {
    prompts.value = prompts.value.filter((prompt) => prompt.date.split('-')[0] === router.currentRoute.value.params.year)
  }
```

This makes fetchPromptsByYear() redundant

Reply:
Exactly!!
# 🎯